src/lib/y2storage/used_storage_features.rb

```diff
@@ -112,8 +112,29 @@ class UsedStorageFeatures
     #----------------------------------------------------------------------
     #
 
-    def initialize(devicegraph)
-      @devicegraph = devicegraph
+    # Initialize object.
+    #
+    # @param arg [Integer or ::Storage::Devicegraph] Either a integer with the feature
+    #   bits or a devicegraph from which to get the feature bits.
+    #
+    def initialize(arg)
+      @devicegraph = nil
+      @used_features = nil
```

Comment:
this initialization is not needed as it is default value in newly created object.

Reply:
I think it helps to understand the code.